global-services-testing/sections/testing/global-broker.adoc

```diff
@@ -11,6 +11,140 @@ A Global Broker:
+
+They will be used during the functional tests of the Global Broker.
+
+===== For the performance tests
+
+Two hundred WIS2 Nodes, only for WIS2 Notification Messages, will be deployed on 5 of the same VMs. On each VM, the equivalent of 40 WIS2 Nodes will be deployed. They will share the same MQTT Broker, however the _centre_id_ and the subscription topic will be different for each WIS2 Node.
+
+The DNS name of the VM will be:
+
+- test-node-1.wis2dev.io (for centre_id from 100 to 139)
+
+- test-node-2.wis2dev.io (for centre_id from 140 to 179)
+
+...
+
+- test-node-5.wis2dev.io (for centre_id from 260 to 299)
+
+===== MQTT Configuration 
+
+The MQTT broker(s) will be reachable using:
+
+```
+MQTT_SUB_BROKER=mqtt://test-node-1.wis2dev.io (and also -2, -3,... -10)
+MQTT_SUB_USERNAME=everyone
+MQTT_SUB_PASSWORD=onlyone
+```
+
+1. Default MQTT port will be used (1883)
+2. The password is changed to _onlyone_. This allows to prevent the default authentication (everyone/everyone) to be usable, while keeping the authorization (typically using only the username) unchanged
+
+===== Notification Message tests
+
+As no metadata record will be provided, checking the existence of the metadata record by the Global Broker must be disabled.
+
+As the topic used for the test messages will be correct, checking the validity of the Topic Hierarchy must be done.
+
+As the Notification Messages will be syntactically correct, checking the compliance of the Notification Message with the defined schema (schemas.wmo.int) must be done.
+
+===== centre_id and subscription
+
+The centre_id will be of the from io-wis2dev-[number]-test with number varying from 11 to 20, and 100 to 299. 100 to 139 on test-node-1.wis2dev.io and so on.
+
+Messages will be published on the topic `origin/a/wis2/io-wis2dev-x-test/#` (with x varying from 11 to 20 and 100 to 299).
+
+===== Summary
+
+Using the antiloop software used by Brazil and France, as an example, the configuration file _io-wis2dev-100-test.env_ will be : 
+
+```
+MQTT_SUB_BROKER=mqtt://test-node-1.wis2dev.io
+MQTT_SUB_USERNAME=everyone
+MQTT_SUB_PASSWORD=onlyone
+MQTT_SUB_TOPIC=origin/a/wis2/io-wis2dev-100-test/#
+CENTRE_ID=io-wis2dev-100-test
+MSG_CHECK_OPTION=verify
+TOPIC_CHECK_OPTION=verify
+METADATA_CHECK_OPTION=no
+GDC_URL=https://api.weather.gc.ca/collections/wis2-discovery-metadata/items?lang=en&f=json&q=
+SCHEMA_URL=https://raw.githubusercontent.com/golfvert/wis2-topic-hierarchy/main/wis2-notification-message.json
+```
+
+The configuration file _io-wis2dev-210-test.env_ will be : 
+
+```
+MQTT_SUB_BROKER=mqtt://test-node-3.wis2dev.io
+MQTT_SUB_USERNAME=everyone
+MQTT_SUB_PASSWORD=onlyone
+MQTT_SUB_TOPIC=origin/a/wis2/io-wis2dev-210-test/#
+CENTRE_ID=io-wis2dev-210-test
+MSG_CHECK_OPTION=verify
+TOPIC_CHECK_OPTION=verify
+METADATA_CHECK_OPTION=no
+GDC_URL=https://api.weather.gc.ca/collections/wis2-discovery-metadata/items?lang=en&f=json&q=
```

Comment:
update to `https://wis2-gdc.weather.gc.ca/collections/wis2-discovery-metadata/items?lang=en&f=json&q=`